### DISCUSSION TITLE
Customizing Material UI for the organizations

### DISCUSSION THREAD

**Discussion by ahmed-asiri** [angular/components#25203] [Q&A]
### Greeting dears,

May I ask if we can clone Material UI (Angular components) and customize the src code of the design system, then we want to have the updated src code on our on-premise gitlab?

The reason, we want to have a design system for our organization, but we don't have enough resources, so we think that we might use already built design system and customize it. We want to apply the organization theme once, and the team install it on the multiple projects.

Is it possible? or is there another way to do that?

Thanks a lot.

**Comment by AirSayy** [comment]
Good day ahmed,

I just saw that your question has not been answered , although I may not have the answer yet to your question. If you haven't already gotten a solution, ill try my best to reply you **asap**.

Cheers.

**Comment by AirSayy** [comment]
You can try if this link answers your question 
https://material.angular.io/
